parsers/UA.py

```diff
@@ -75,10 +79,21 @@ def fetch_production(
                 row["production"][v] = 0.0
```

Comment:
While not related to your changes we have a standardized way to do this now with the new EventList classes.

You would need to import and use the following.
from electricitymap.contrib.lib.models.event_lists import (
    ProductionBreakdownList,
)
from electricitymap.contrib.lib.models.events import (
    ProductionMix,
    StorageMix,
)

Reply:
If you can, please, go ahead and make changes in my PR. I tried to compare it to other parses to get a grasp of how it should be used, but I'm not sure, so, I think it will be better if you do it.

Reply:
Will do so tomorrow.